src/modules/75-cd/components/PipelineSteps/TerraformPlan/TfPlanVarFileList.tsx

```diff
@@ -42,10 +42,12 @@ interface TfVarFileProps {
   formik: FormikProps<TFPlanFormData>
   isReadonly?: boolean
   allowableTypes: MultiTypeInputType[]
+  getNewConnectorSteps?: any
+  setSelectedConnector?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232256)
```suggestion
  setSelectedConnector?: unknown
```